lib/launcher.ts

```diff
@@ -251,7 +254,7 @@ let initFn = function(configFile: string, additionalConfig: Config) {
         let createNextTaskRunner = () => {
           let task = scheduler.nextTask();
           if (task) {
-            let taskRunner = new TaskRunner(configFile, additionalConfig, task, forkProcess);
```

Comment:
Does this break setting flags on the command line? It looks like cli.ts passes in `argv` as `additionalConfig`.

I think the reason additionalConfig is passed into the task runner is so that it can be the last thing merged into the config, after the multicapabilities logic and any other files.

I like moving config parsing out of the TaskRunner, though. Maybe add the additionalConfig right before this?